kong/plugins/rate-limiting/policies/redis-cluster.lua

```diff
@@ -0,0 +1,136 @@
+local timestamp = require "kong.tools.timestamp"
+-- TODO add this to dependency BLOCKER (need to publish https://github.com/steve0511/resty-redis-cluster on luarocks)
+local redis_cluster = require "rediscluster" 
```

Comment:
This require will fail. https://github.com/steve0511/resty-redis-cluster is the module being referred here.